components/ws-manager-api/go/crd/v1/workspace_types.go

```diff
@@ -250,6 +250,10 @@ type WorkspaceList struct {
 	Items           []Workspace `json:"items"`
 }
 
+func (w *Workspace) IsHeadless() bool {
```

Comment:
Consider adding a comment to describe why? Other than that, :shipit: . :smile:

Reply:
Added